code/modules/shuttle/shuttle.dm

```diff
@@ -163,8 +163,6 @@
 	var/turf/T1 = locate(L[3],L[4],z)
 	for(var/turf/T in block(T0,T1))
 		T.color = _color
-		//LAZYINITLIST(T.atom_colours)
-		T.maptext = null
```

Comment:
sorry, i meant just the comment, not the line below it. this is important

Reply:
i didn't mean to remove it